addons/mail/static/src/components/call_view/call_view.js

```diff
@@ -109,7 +108,7 @@ export class CallView extends Component {
             tileCount: this.tileContainerRef.el.children.length,
         });
 
-        this.state.tileWidth = tileWidth;
+        this.callView._update({ tileWidth });
```

Comment:
Why do you use the private update here?

Reply:
Typo, will fix it.
IIRC, I was fighting with the autocompletion feature of IDE, and it still let me type the wrong things 🥲

Reply:
Changes done